src/os_linux.cpp

```diff
@@ -351,7 +351,7 @@ u64 OS::getTotalCpuTime(u64* utime, u64* stime) {
     }
 
     u64 real = (u64)-1;
-    char buf[512];
+    char buf[512] = {0};
```

Comment:
Can you shrink buffer to the minimum required size? I think 128 will be good.
It did not make difference for uninitialized buffer, but should reduce amount of work for zeroing.